tests/fixtures.py

```diff
@@ -298,7 +298,8 @@ def finalizer():
     class_instance.dc_pod_objs = [
         helpers.create_pod(
             interface_type=class_instance.interface, pvc_name=pvc_obj.name, do_reload=False,
```

Comment:
do we still need this fixture? maybe it would be better to use the new factory fixture for deployment config

Reply:
This is an old fixture if someone needs they can use this but for my script, I am using factory fixture